python/fledge/common/configuration_manager.py

```diff
@@ -1995,3 +1995,9 @@ def _validate_min_max(_type, val):
                         if not isinstance(eval_s, type_check):
                             raise ValueError(type_mismatched_message)
 
+    def _update_special_config_items(self, cat_name: str, cat_value: dict) -> None:
+        """ Update value in config items for a category which will be required without restarting fledge """
```

Comment:
which will not/don't require a restart of Fledge
or
which are required without restart of Fledge

Ideally, We don't want a restart anyway.